src/administrator/models/forms/content.xml

```diff
@@ -1,38 +1,35 @@
+      <option value="1">JPUBLISHED</option>
+      <option value="0">JUNPUBLISHED</option>
+      <option value="2">JARCHIVED</option>
+      <option value="-2">JTRASHED</option>
+    </field>
+    <field name="checked_out" type="hidden" filter="unset" />
+    <field name="checked_out_time" default="0000-00-00 00:00:00" type="hidden" filter="unset" />
+    <field name="title" required="true" label="COM_TC_FORM_LBL_CONTENT_TITLE" description="COM_TC_FORM_DESC_CONTENT_TITLE" hint="COM_TC_FORM_LBL_CONTENT_TITLE" />
+    <field name="version" id="version_id" required="true" class="validate-version" label="COM_TC_FORM_LBL_CONTENT_VERSION" description="COM_TC_FORM_DESC_CONTENT_VERSION" hint="COM_TC_FORM_LBL_CONTENT_VERSION" />
+    <field name="client" required="true" id="client_id" class="validate-client" label="COM_TC_FORM_LBL_CONTENT_CLIENT" description="COM_TC_FORM_DESC_CONTENT_CLIENT" hint="COM_TC_FORM_LBL_CONTENT_CLIENT" />
+    <field name= "url_pattern"  id="url_pattern_id" description= "COM_TC_FORM_LBL_CONTENT_URL_PATTERN_DESC"  type= "subform"  label= "COM_TC_FORM_LBL_CONTENT_URL_PATTERN"  min= "1"  max= "1000"  formsource= "/administrator/components/com_tc/models/forms/url_pattern.xml"  multiple= "true"  buttons= "add,remove"  layout="joomla.form.field.subform.repeatable-table" groupByFieldset="false" />
+    <field name="start_date" type="calendar" default="now" showtime="true" class="inputbox" format="%Y-%m-%d %H:%M:%S" required="true" label="COM_TC_FORM_LBL_CONTENT_START_DATE" description="COM_TC_FORM_DESC_CONTENT_START_DATE" hint="COM_TC_FORM_LBL_CONTENT_START_DATE" filter="user_utc" />
+    <field name="content" type="editor" filter="raw" label="COM_TC_FORM_LBL_CONTENT_CONTENT" buttons="true" description="COM_TC_FORM_DESC_CONTENT_CONTENT" hint="COM_TC_FORM_LBL_CONTENT_CONTENT" />
+    <field name="global" type="radio" class="btn-group" id="global_id" label="COM_TC_FORM_LBL_CONTENT_GLOBAL" description="COM_TC_FORM_LBL_CONTENT_GLOBLA_DESC" default="0">
+      <option value="1">JYES</option>
+      <option value="0">JNO</option>
+    </field>
+    <field name="groups" type="usergrouplist" multiple="true" default="0" label="COM_TC_FORM_LBL_CONTENT_SELECT_USER_GROUP" description="COM_TC_FORM_LBL_CONTENT_SELECT_USER_GROUP_DESC" />
+    <field name="is_blacklist" type="radio" class="btn-group" label="COM_TC_FORM_LBL_CONTENT_USER_GROUP_BEHAVIOUR" description="COM_TC_FORM_LBL_CONTENT_USER_GROUP_BEHAVIOUR_DESC" default="1">
+      <option value="1">COM_TC_FORM_LBL_CONTENT_USER_GROUP_BEHAVIOUR_YES</option>
+      <option value="0">COM_TC_FORM_LBL_CONTENT_USER_GROUP_BEHAVIOUR_NO</option>
+    </field>
+    <field name="enforce_delay_days" type="hidden" required="" label="COM_TC_FORM_LBL_CONTENT_ENFORCE_DELAYS_DAY" description="COM_TC_FORM_LBL_CONTENT_ENFORCE_DELAYS_DAY_DESC" hint="COM_TC_FORM_LBL_CONTENT_ENFORCE_DELAYS_DAY" />

```

Comment:
this can be a number field instead of text/hidden (in future maybe)?

Reply:
Currently, we are not using this field hence marked as hidden